ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -1810,8 +1810,8 @@ tests:
     - ref: idp-htpasswd
     - ref: openshift-extended-test-longduration
     workflow: cucushift-installer-rehearse-azure-ipi-private
-- as: azure-ipi-marketplace-f14
```

Comment:
Do you mind to add one more destructive ci jobs for azure marketplace together in this PR?

Reply:
seem like azure get marketplace supported from 4.11

Reply:
Added

Reply:
Great, thanks.